python/pyspark/sql/__init__.py

```diff
@@ -41,6 +41,13 @@
 """
 from __future__ import absolute_import
 
+
+def since(version):
```

Comment:
Ah, gotcha.  Just asking because #6275 adds a missing `startTime` property to PySpark's context and I was thinking about suggesting that we add the `@since` tag there.